src/formatter/ExpressionFormatter.ts

```diff
@@ -286,7 +286,7 @@ export default class ExpressionFormatter {
   }
 
   private formatIdentifier(node: IdentifierNode) {
-    this.layout.add(node.text, WS.SPACE);
+    this.layout.add(this.showIdentifier(node), WS.SPACE);
```

Comment:
In addition to plain identifiers we also have several identifier-like things:

- variables
- parameters

these usually behave just like identifiers and the only thing distinguishing them from normal identifiers is some prefix like `@myvar` or `:myparam`.

As the parser currently treats variables as identifiers, these too end up upper/lowercased. But parameters are kepts separate by the parser, so the case of these doesn't change. Should parameters also change together with identifiers?

Reply:
My change is only for unquoted identifiers - I think variables and parameters are outside the scope and should be handled in another PR...